dali/base/dautils.cpp

```diff
@@ -111,6 +111,28 @@ IPropertyTree * getDropZonePlane(const char * name)
     xpath.appendf("[@name='%s']", name);
     return getGlobalConfigSP()->getPropTree(xpath);
 }
+IPropertyTree * findDropZonePlane(const char * path, const char * host, bool ipMatch)
```

Comment:
trivial/not new: unusual not to have new line between implementation methods in a cpp.
Can you add and to ones that proceed this new method?